packages/circuit-ui/components/ColorInput/ColorInput.stories.tsx

```diff
@@ -32,3 +32,9 @@ export const Base = (args: ColorInputProps) => (
 );
 
 Base.args = baseArgs;
+
+export const Readonly = (args: ColorInputProps) => (
+  <ColorInput {...args} style={{ maxWidth: '250px' }} />
```

Comment:
The `maxWidth` doesn't seem necessary here and for the base story as it barely changes the width of the input. Probably some copy-pasta when this component was created – let's remove it.